lib/object.c

```diff
@@ -19,14 +15,6 @@
 TPM2B_PRIVATE tpm2_util_object_tsspem_priv = { 0 };
 TPM2B_PUBLIC tpm2_util_object_tsspem_pub = { 0 };
 
-typedef struct {
-    ASN1_OBJECT *type;
-    ASN1_BOOLEAN emptyAuth;
-    ASN1_INTEGER *parent;
-    ASN1_OCTET_STRING *pubkey;
-    ASN1_OCTET_STRING *privkey;
-} TSSPRIVKEY_OBJ;
-
```

Comment:
So you can include object.h in **tpm2_encodeobject** and remove https://github.com/tpm2-software/tpm2-tools/blob/master/tools/misc/tpm2_encodeobject.c#L16:#L52 ?

Reply:
You can add a commit to this PR.